recipes/cime-env/meta.yaml

```diff
@@ -30,12 +30,12 @@ requirements:
     - pandas
     - matplotlib
     - jinja2
-    - json_tricks 3.11.0
+    - json_tricks 3.15.5
     - pybtex >=0.21
     - livvkit 2.1.6
-    - hdf5 1.10.5
-    - netcdf4 1.5.1.2
-    - libnetcdf 4.6.2
+    - hdf5 1.10.6
+    - netcdf4 1.5.6
```

Comment:
@jhkennedy, same question with `netcdf4`.

Reply:
Really, as far as EVV goes, you could probably only list `evv4esm`, but maybe some of the minimum pinings here should be added to EVV's recipe: https://github.com/conda-forge/evv4esm-feedstock/blob/master/recipe/meta.yaml#L24-L32

I think the only deps outside of EVV would be these two:
https://github.com/ESMCI/cime/blob/000b3c7d5b28e1e2865c95c681a6e5ae23ccde28/scripts/lib/CIME/SystemTests/pgn.py#L21-L22
* Pandas
* Numpy

Reply:
And maybe the full matplotlib instead of just matplotlib-base. 

I don't remember if some of these dependencies were added as general "cime testing would want these" but, in general, I'm inclined to restrict to what's being *used* and add additional things upon request